src/endpoints/Branches.js

```diff
@@ -30,10 +30,10 @@ export default class Branches extends Endpoint {
 
       cli: async () => {
         const response = await this.cliRequest([
-          "branch",
-          "load",
-          descriptor.projectId,
-          descriptor.branchId
+          "branches",
+          "get",
+          descriptor.branchId,
+          ...["--project-id", descriptor.projectId]
```

Comment:
Since we know that a project ID will always be passed, we don't need to spread an array. Instead, we can just list `"--project-id", descriptor.projectId`. This comment applies throughout and is definitely nonblocking.

```suggestion
          "--project-id",
          descriptor.projectId
```